x-pack/libbeat/management/config.go

```diff
@@ -0,0 +1,74 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package management
+
+import (
+	"os"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/common/file"
+	"github.com/elastic/beats/libbeat/kibana"
+	"github.com/elastic/beats/libbeat/paths"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+// Config for central management
+type Config struct {
+	// true when enrolled
+	Enabled bool
+
+	AccessToken string
+
+	Kibana *kibana.ClientConfig
+
+	Configs []struct {
+		Name   string
+		Config *common.Config
+	}
+}
+
+// Load settings from its source file
+func (c *Config) Load() error {
+	path := paths.Resolve(paths.Data, "management.yml")
```

Comment:
As you said, this file stores internal data for central management, like kibana endpoint settings or cache of retrieved configs. It's not meant to be edited by the user

Reply:
What about naming the object `Management`? Perhaps it's just me but when I read `Config` and see unpack, I'm thinking of our `beat.yml` file :-)

Reply:
I plan to add a Management/Manager object, as this is `management.Config` I think the namespace clarifies it. What about renaming it to `Settings` to avoid the `Config` term?

Reply:
It could help but let's discuss it again when you add the management/manager object.